gulpfile.js

```diff
@@ -86,7 +86,7 @@ function write(streams, name) {
   return concat(function(body) {
     return file(dest, body, {src: true})
       .pipe(rev())
-      .pipe(gulpif(production, uglify()))
+      .pipe(uglify())
```

Comment:
I believe FEC needs to be double checked because I think it's not being uglified on local for that as well.